src/JBERC20.sol

```diff
@@ -68,6 +77,25 @@ contract JBERC20 is ERC20Votes, ERC20Permit, Ownable, IJBToken {
         return _burn(account, amount);
     }
 
+    //*********************************************************************//
+    // ----------------------- public transactions ----------------------- //
+    //*********************************************************************//
+
+    /// @notice Initialized the token.
+    /// @param name_ The name of the token.
+    /// @param symbol_ The symbol that the token should be represented by.
+    /// @param owner The owner of the token.
+    function initialize(string memory name_, string memory symbol_, address owner) public override {
+        // Stop re-initialization.
+        if (this.owner() != address(0) || owner == address(0)) revert();
```

Comment:
if the owner calls Ownable::renounceOwnership(), then anyone can re-initialise it - rn it's ok as it should always be JBTokens the owner, but isn't a bit smelly for the long run? Why not just checking if the name/symbol are already set instead (there is no "clearNameAndSymbol()")

Reply:
Good idea!